src/main/java/nextstep/subway/line/dto/LinesResponse.java

```diff
@@ -0,0 +1,29 @@
+package nextstep.subway.line.dto;
+
+import nextstep.subway.line.domain.Line;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class LinesResponse {
+    private List<LineResponse> lines;
+
+    public LinesResponse() {
+        this.lines = new ArrayList<>();
```

Comment:
```suggestion
        this(new ArrayList<>());
```

인스턴스 변수의 초기화는 하나의 생성자에서 해주는 것이 좋은데요
어떠한 차이가 있을지 고민해보면 좋겠어요!

Reply:
리뷰 감사합니다. 이렇게 사용하는 것은 처음보는것 같아요 .!
관련 자료를 어떻게 찾아볼 수 있을까요?